### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by JakariaJishan**
- Use webpack to bundle JavaScript.
- use proper ES6 syntax.
- Use ES6 modules to write modular JavaScript.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,30 +1,96 @@
...
+  todos[newArr].desc = value;
+  localStorage.setItem('todos', JSON.stringify(todos) || []);
+};
+
+// remove item
+
+const removeTodo = (id) => {
+  const todos = JSON.parse(localStorage.getItem('todos'));
+  const newTodos = todos.filter((item) => item.index !== id);
+  // newTodos.map((elem, index) => newTodos[index].id = index + 1);
```
Comment (Whoistolu) [inline src/index.js]: - [x] [OPTIONAL] Let's remove the unused/commented code so that we will have a clean code base.

File: src/index.js [hunk 2]
```diff
@@ -1,30 +1,96 @@
...
+};
+
+// update todo
+
+const updateTask = (value, id) => {
+  const todos = JSON.parse(localStorage.getItem('todos'));
+  const newArr = todos.findIndex((elem) => elem.index === id);
+  todos[newArr].desc = value;
+  localStorage.setItem('todos', JSON.stringify(todos) || []);
+};
```
Comment (Whoistolu) [inline src/index.js]: - [x] [REQUIRED] Kindly ensure you add a function for editing task descriptions. Right now, you have all functions working properly except the edit function which is absent, and the localStorage gets updated. Let's implement this.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @JakariaJishan 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @JakariaJishan, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations

## Good Highlights✔️

- Requested changes have been implemented.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**